landa/water_body_management/report/catch_log_statistics/catch_log_statistics.py

```diff
@@ -66,60 +39,104 @@
 		"fieldtype": "Float",
 		"label": "Weight in Kg",
 	},
+	{
+		"fieldname": "by_foreign_regional_org",
+		"fieldtype": "Percent",
+		"label": "By Foreign Regional Organization",
```

Comment:
Perhaps this can be more clear as "No. of Fish by Foreign Regional Organization" or place this column next to No. of fish. To me I wondered if the percentage was pertaining to the weight.
If this works for our client then feel free to ignore to my suggestion.